### PR TITLE
feat: list model components on individual integration pages

### LINKED ISSUE #5282: Incorporation of the Model Components into each integrtion page as described using make.com as the example.
### Environment:
- Host OS:
- Browser:

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by vishalvivekm**
**Description**

Currently the components lack the description, so it makes sense to list the Icon with just the component's name.

image

This PR fixes #5282 

**Notes for Reviewers**
### This PR needs enhancements in terms of styling..

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 6a580f5d28526496fa37ad8011a54154ff2b60cc at: https://65bd4001819da43028d891f6--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 6a580f5d28526496fa37ad8011a54154ff2b60cc at: https://65bd40113889392ef8a51bd8--layer5.netlify.app

**Comment by vishalvivekm** [comment]
@vishalvivekm 
Let's discuss this on website call at 6:30 PM IST / 7 AM Central time Today. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima).

**Comment by hadihaider055** [comment]
@vishalvivekm could you please assign it to me? I will update the components section, with the changes discussed by @leecalcote.

**Comment by l5io** [comment]
🚀 Preview for commit 2d10bda4059d698ccfd107a8e4584effcb3f460c at: https://65cbcb3273b8121a67824af4--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit d0258a932b899ce3fa7ccf71e35e2b872f8e581d at: https://65cbce00136e021f45d53684--layer5.netlify.app

**Comment by leecalcote** [comment]
Ping when the `do-not-merge` is no longer tagged here.

**Comment by vishalvivekm** [comment]
Thanks a bunch @ritiksaxena124 

Here's the preview after the enhancement Ritik's done:

Screenshot from 2024-02-14 01-58-38

**Comment by leecalcote** [comment]
Much better. A `border-radius: .5rem` might not hurt.

File: src/sections/Meshery/Meshery-integrations/Individual-Integrations/Component.style.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    padding: 3rem 2rem 5rem 2rem;
+    justify-content: center;
+  }
+  .maincontainer {
+    display: flex;
+    align-items: center;
+    gap: 1rem;
+    background-color: ${props => props.theme.grey212121ToGreyEEEEEE};
+    padding: 0.5rem 1rem;
+    border-radius: 0.25rem;
```
Comment (leecalcote) [inline src/sections/Meshery/Meshery-integrations/Individual-Integrations/Component.style.js:39]:
```suggestion
    border-radius: 0.85rem;
```

Comment (leecalcote) [reply src/sections/Meshery/Meshery-integrations/Individual-Integrations/Component.style.js:39]: Just a guess that this size might fit a bit better...

Comment (vishalvivekm) [reply src/sections/Meshery/Meshery-integrations/Individual-Integrations/Component.style.js]: commited.

**Comment by leecalcote** [comment]
What sort of empty state do we have here?

**Comment by vishalvivekm** [comment]
> What sort of empty state do we have here?

if an integration doesn't have any component, we just skip displaying this section

**Comment by leecalcote** [comment]
the ENTIRE section, right? Good.

**Comment by vishalvivekm** [comment]
> the ENTIRE section, right? Good.

Yes, right.

**Comment by leecalcote** [comment]
Not a section heading with no component boxes...

**Comment by leecalcote** [comment]
Ooooh, I'm just noticing the verbiage used. I'll make an update...

**Comment by leecalcote** [comment]
Good stuff here, though, @vishalvivekm 💯

**Pull request merged**